ui/src/components/Search.js

```diff
@@ -324,6 +325,29 @@ class Search extends React.Component {
       );
     };
 
+    // Use react-window to make UI faster when there are a lot of search
+    // results. See
+    // https://github.com/JedWatson/react-select/issues/2850#issuecomment-410318717
+    const height = 35;
```

Comment:
Does this need to be out here? Shouldn't it go inside the MenuList's render method?

Reply:
Done